README.md

```diff
@@ -4,10 +4,27 @@ This repository contains the yaml files used for the release workstream HACBS mi
 
 These yaml files assume you have bootstrapped a working appstudio configuration from [infra-deployments](https://github.com/redhat-appstudio/infra-deployments).
 
-## Tags
+## Searching for previous demos
 
 Milestones are separated in this repository by tags.
 To view the demo resources for a previous milestone, simply checkout its tag.
 Be warned, however, that we make no guarantee that the past demo resources work with the latest HACBS deployment.
 Once a milestone passes, we will only be focusing on demo resources for the next milestone.
 So, if you need working demo resources, use the `main` branch.
+
+## Running the current demo (milestone 5)
```

Comment:
I'd probably remove any mention to the milestone. Ideally, with time, the `README.md` shouldn't need a change per milestone.